### ISSUE TITLE
sometimes owl bot doesn't run

### ISSUE THREAD

**Issue by SurferJeffAtGoogle** [googleapis/repo-automation-bots#2204]
On bilge pump duty this morning, I've already seen 5 pull requests where owl bot never ran, so I couldn't merge them.  

For example: https://github.com/googleapis/nodejs-service-management/pull/63

These pull requests would already have merged if not for owl bot, so this is slowing us down.

**Comment by tmatsuo** [comment]
Do we know why? Were there a burst of webhook requests at the time when the PR is created?

**Comment by SurferJeffAtGoogle** [comment]
More examples:
https://github.com/googleapis/nodejs-dlp/pull/629
https://github.com/googleapis/nodejs-contact-center-insights/pull/3

**Comment by SurferJeffAtGoogle** [comment]
@bcoe has some ideas why.

**Comment by bcoe** [comment]
So far, whenever I've dug into these issues, I've found a corresponding set of timeouts on GitHub WebHooks as described here:

https://github.com/googleapis/repo-automation-bots/issues/2114

I think once we address #2114, we might notice that failures to launch the post processor are rare.

**Comment by SurferJeffAtGoogle** [comment]
More examples:
https://github.com/googleapis/nodejs-retail/pull/67
https://github.com/googleapis/nodejs-analytics-admin/pull/132
https://github.com/googleapis/nodejs-life-sciences/pull/11

**Comment by SurferJeffAtGoogle** [comment]
More examples:
https://github.com/googleapis/nodejs-iam-credentials/pull/66
https://github.com/googleapis/nodejs-analytics-data/pull/122

**Comment by bcoe** [comment]
@SurferJeffAtGoogle we found a bug on some of the repos you linked, which was us failing to report a failed status check when config was invalid (_we had slightly off config in some repos_).

I believe the other failures to run were a result of us missing Webhooks when traffic is bursty, @tmatsuo has addressed this by splitting apart our frontend for OwlBot from our backend job processing. We've tested and have been able to handle 80 jobs concurrently, without any failures.
